jib-gradle-plugin/README.md

```diff
@@ -469,6 +470,12 @@ jib {
 }
 ```
 
+
+### Jib Extensions
+
+The Jib plugins have an extension framework that enables anyone to easily extend and tailor the Jib behavior to their liking. We have already written a few useful extensions, for example, that enables [fine-grained layer control](https://github.com/GoogleContainerTools/jib-extensions/tree/master/first-party/jib-layer-filter-extension-gradle) (such as including deleting files and moving files into new layers), and [supports Quarkus](https://github.com/GoogleContainerTools/jib-extensions/tree/master/first-party/jib-quarkus-extension-gradle). You can also write your own extension. Check out the [jib-extensions](https://github.com/GoogleContainerTools/jib-extensions) repository.
```

Comment:
Not sure fond of the idea of repeating what's in the faq here. Maybe we can redirect to it?

Reply:
The point of #2986 is that some features like fine-grained layer control are worth mentioning in the main plugin doc so that it can be searched and found easily. (The FAQ was an afterthought. I think the main entry for first time users are the main doc, not FAQ.) I still want to mention a few specific extensions, and then it basically becomes the entire FAQ description, which is actually fairly short.

Reply:
sounds good to me.